clarifai.go

```diff
@@ -0,0 +1,245 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"os"
+	"sort"
+	"strings"
+)
+
+const (
+	clarifaiApi = "https://api.clarifai.com/v2/models/aaa03c23b3724a16a56b629203edc62c/outputs"
+	clarifyAuth = "https://api.clarifai.com/v1/token"
+	minValue    = 0.72
+	maxTags     = 10
+)
+
+type ClarifaiClient struct {
+	Token  string
+	Expiry int
+	Access bool
```

Comment:
Access is just a flag that signals whether the client still has access to the api. I just put it there for now, but this is still a work-in-progress (we need to write something that monitors the expiration and requests a renewal only when needed)